src/components/Router/index.js

```diff
@@ -227,6 +229,14 @@ class HistoryRouter extends Component {
     history.push(path, historyState)
   }
 
+  localization = () => {
```

Comment:
not sure if I would put it on the router. it's not really routing. maybe where the redux store is being prepped.

Reply:
also an interesting question to consider if we each component should initialize its own polygot or if it should come from its root

Reply:
Yes, I put it in the router for now just to have it working but probably it should be at a lower level.
What would be the benefit of initialising polyglot in each component?